pkg/installer/installer.go

```diff
@@ -40,8 +41,9 @@ const (
 
 	// config file fields - contain path delimiters for plugin interpretation of config manifest
 	SkipNamespaceDeletionConfig   = "spec.skipNmespaceDeletion"
```

Comment:
also just noticed this - not part of this PR - but should this be `spec.skipNamespaceDeletion` ?